disktools.py

```diff
@@ -1295,3 +1295,76 @@ def getMdNodes():
     except IOError:
         pass
     return nodes
+
+
+class DeviceMounter:
+    class Mount:
+        def __init__(self, dev, mountpoint, options=None, fstype=None):
+            self.dev = dev
+            self.mountpoint = mountpoint
+            self.options = options
+            self.fstype = fstype
+
+        def __repr__(self):
+            return "Mount(%s, %s, %s, %s)" % (self.dev, self.mountpoint, self.fstype, self.options)
+
+        def mount(self):
+            util.mount(self.dev, self.mountpoint, self.options, self.fstype)
+
+        def umount(self):
+            util.umount(self.mountpoint)
+
+    mounts = []
+
+    @classmethod
+    def addMountPoints(cls, arg_mounts):
+        mounts = []
+        for m in arg_mounts:
+            m = m.split(':')
+            if len(m) < 2:
+                continue
+            dev = m[0]
+            fstype = None
+            options = None
+            if len(m) >= 4:
```

Comment:
Should these cases match the specific number of expected elements and fail if there are more than expected?
`if len(m) == 4: ....` ending with some `else: raise...` to show that input wasn't valid?

Reply:
It was to keep compatibility with the previous code. But I suppose it was done for simplicity only. I'll change it